### PR TITLE
New Changes Updated

### PR DESCRIPTION
**Pull request by SamadK01**
Check my code and please reviewed it

### CODE REVIEW

File: logout.php [hunk 1]
```diff
@@ -4,4 +4,6 @@
 echo "Faaiz uddin";
 echo "Rabsha";
 echo "Najam";
+echo "edit by samad";
```
Comment (Faaiz-uddin) [inline logout.php:7]: can you explain it why this changes is needed

Comment (SamadK01) [reply logout.php:7]: yes beacuse some changes need

**Review by Faaiz-uddin — APPROVED** [review]
